src/writer/proposal.ts

```diff
@@ -64,6 +65,20 @@ export async function verify(body): Promise<any> {
     return Promise.reject(`invalid space settings: ${e}`);
   }
 
+  try {
+    const content = `
+      ${msg.payload.name || ''}
+      ${msg.payload.body || ''}
+      ${msg.payload.discussion || ''}
+    `;
+
+    if (await isMalicious(content)) {
+      return Promise.reject(`invalid proposal content`);
```

Comment:
Maybe 
```suggestion
      return Promise.reject(`malicious proposal content`);
```

Reply:
I'd rather stay vague, we may catch scammer asking in our support why his proposal is rejected.